mne/io/kit/kit.py

```diff
@@ -208,6 +208,7 @@ def _set_stimchannels(self, info, stim, stim_code):
             chan_info['loc'] = np.zeros(12)
             chan_info['kind'] = FIFF.FIFFV_STIM_CH
             info['chs'].append(chan_info)
+            info._update_redundant()
```

Comment:
no, it only needs to be called (again) if `chs` is updated and it's not updated if `stim is None`